recipes/opencv/4.x/test_package/android_profile

```diff
@@ -0,0 +1,15 @@
+include(default)
```

Comment:
I believe this file can be remove

Reply:
agree, not needed

Reply:
Yeah, I'll remove it. I was not sure how to deal with files used for builds other than LInux/Windows/MacOS. I guess we don't keep them at the moment?

Reply:
From what I can tell, they are built into C3i system such that every recipe uses the same profiles

Reply:
But none for Android and iOS of course, right?

Reply:
macOS is supported but no not android